### DISCUSSION TITLE
Electrification of Household Heating (Project 201)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#31] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #201](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Electrification%20of%20Household%20Heating)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by ghost** [comment]
"Hi, I am a student at UOM and interested in working and collaborating on this project

**Comment by hiedt** [comment]
Hi @robertogl. The topic suggests comparing two electrical heating systems, but it only mentions some actuators (i.e., radiator, heat pump). Does it mean that I should use the same controller (e.g., bang-bang) to benchmark two actuators? Or should I compare two control algorithms on the same actuator instead? 

I am not sure which one is more favorable since, for example, a well-designed MPC on a traditional radiator might outperform a naive bang-bang on a bulky heat pump. Thanks.

**Reply by robertogl** [reply]
Hi @hieudtrung,

Both analysis would actually be interesting to see. In fact, to have a thorough comparison of systems with different actuators I would initially use a common control algorithm for both actuators so to provide a baseline comparison of the actuators' inherent efficiencies and performance characteristics. However, it is important to note that the best controller design often depends on the specific actuator being controlled. Therefore, ideally, the comparison should be made between different systems where each actuator is paired with the control algorithm that best suits it.

I hope it makes sense.
Thank you.
